toolkits/landcover/ee_repo_sync.py

```diff
@@ -69,10 +75,12 @@ def search_replace(start_path, ext, original, replacement):
     new_require_path += target_path + '/'
 
 # Clone or pull repo to home dir
-if not os.path.isdir(clone_path):
-    subprocess.call(['git', 'clone', repo_url, clone_path])
+if os.path.isdir(clone_path):
+    print('%s exists. Pulling latest from remote.' % clone_path)
+    subprocess.call(['git', '-C', clone_path, 'pull', '--quiet'])
```

Comment:
I suggest you switch `subprocess.call` to `subprocess.check_output` for all shell calls.  That will make sure trouble causes an exception rather than moving on.

- https://docs.python.org/library/subprocess.html#subprocess.check_output
- https://stackoverflow.com/questions/4917871/does-git-return-specific-return-error-codes

Reply:
Done. I needed to check if there were files to commit before calling `git commit` since it fails with 1 when there's nothing to commit.